src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -368,12 +403,12 @@ private enum NamespaceParts
             TopLevelStatementsAfterTypesAndNamespaces = 6,
```

Comment:
don't we need to modify `NamespaceParts`?

Reply:
in what way?

Reply:
If we allow the same ordering of regular namepsaces as you clarified in the other thread we probably don't need to.